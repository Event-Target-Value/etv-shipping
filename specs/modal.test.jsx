import React from 'react';
import { shallow } from 'enzyme';
import Modal from '../client/src/components/ModalOne.jsx';


describe('<Modal />', () => {
  it('Modal functional and correctly rendering', () => {
   const wrapper = shallow(<Modal />);
   expect(wrapper).toMatchSnapshot();
  });


  it('Displays current product', () => {
    const wrapper = shallow(<Modal/>);
    expect()
  })
});
