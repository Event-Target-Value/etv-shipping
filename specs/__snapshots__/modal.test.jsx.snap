// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Modal /> Modal functional and correctly rendering 1`] = `""`;
